### PR TITLE
@damassi => [ArtistPage] Auction Result Item component

### PR DESCRIPTION
**Pull request by mzikherman**
First stab, the responsiveness feels a bit off when `xs` breakpoint is hit. Like, that's going to come in on sizes < 767px (if I am reading https://github.com/artsy/palette/blob/44b4c30e63802be43cc97b881d25095daf8b6d34/src/Theme.tsx#L56 right), but the comp seems to have the thumbnail (taking up one column) as being 30px wide? That feels too small...

bp

### CODE REVIEW

File: src/Styleguide/Components/AuctionResultItem.tsx [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+}
+
+export class AuctionResultItem extends React.Component<AuctionResultItemProps> {
+  render() {
+    return (
+      <Responsive>
+        {({ xs, sm }) => {
+          if (xs) return <ExtraSmallAuctionResultItem {...this.props} />
+          else if (sm) return <SmallAuctionResultItem {...this.props} />
+          return <LargeAuctionResultItem {...this.props} />
```
Comment (damassi) [inline src/Styleguide/Components/AuctionResultItem.tsx]: @mzikherman - It looks like there's `@l`, `@m` and `@s`[for this page](zpl://project?pid=5b113f06463238c438431ea3).

Regarding your question in the description, i think at the small breakpoint the left column stops growing and is fixed at a fixed width (slightly larger than the small image), and the middle / right columns stretch.

File: src/Styleguide/Components/AuctionResultItem.tsx [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+    </Flex>
+    <Flex flexDirection="column" justifyContent="space-between">
+      <div>
+        <Serif size="3t">Sale: {props.salePrice}</Serif>
+        <MetadataContainer>
+          <Serif size="3t">Est: {props.estimate}</Serif>
+        </MetadataContainer>
+      </div>
+    </Flex>
+  </Flex>
```
Comment (damassi) [inline src/Styleguide/Components/AuctionResultItem.tsx]: easy as pie 👍

File: src/Styleguide/Components/AuctionResultItem.tsx [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+          height="70px"
+          mr={4}
+          ml={4}
+        />
+      </div>
+    </Flex>
+    <Flex flexDirection="column" justifyContent="space-between">
+      <div>
+        <Serif size="3t">
+          <Title>{props.title}, </Title>
```
Comment (damassi) [inline src/Styleguide/Components/AuctionResultItem.tsx]: Should this be `<Serif size='3t' italic>{props.title}</Serif>`?

Comment (mzikherman) [reply src/Styleguide/Components/AuctionResultItem.tsx]: Well, `Title` was using a `span` for italics to get the title, date on one line. Versus separate `Serif` tags for each, both are divs/blocks.

Comment (damassi) [reply src/Styleguide/Components/AuctionResultItem.tsx]: Ah, if you want you can add `display='whatever'`

File: src/Styleguide/Components/AuctionResultItem.tsx [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+export const ExtraSmallAuctionResultItem = (props: AuctionResultItemProps) => {
+  return (
+    <Flex width="100%" justifyContent="space-between">
+      <Flex width="70%">
+        <div>
+          <ScaledArtworkImage
+            src={props.imageUrl}
+            width="auto"
+            height="30px"
+            mx={4}
```
Comment (damassi) [inline src/Styleguide/Components/AuctionResultItem.tsx]: I'm gonna go through and annotate the new values based on master:

2

File: src/Styleguide/Components/AuctionResultItem.tsx [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+        <div>
+          <Serif size="2">
+            <Title>{props.title}, </Title>
+            {props.date}
+          </Serif>
+          <MetadataContainer>
+            <Serif size="2">{props.dimensions.in}</Serif>
+            <Serif size="2">{props.dimensions.cm}</Serif>
+          </MetadataContainer>
+          <Box pt={3}>
```
Comment (damassi) [inline src/Styleguide/Components/AuctionResultItem.tsx]: 1

File: src/Styleguide/Components/AuctionResultItem.tsx [hunk 6]
```diff
@@ -0,0 +1,183 @@
...
+            <Serif size="2">{props.organization}</Serif>
+            <MetadataContainer>
+              <Serif size="2">{props.auctionDate}</Serif>
+              <Serif size="2">
+                <TextLink underline>View Details</TextLink>
+              </Serif>
+            </MetadataContainer>
+          </Box>
+
+          <Box pt={3}>
```
Comment (damassi) [inline src/Styleguide/Components/AuctionResultItem.tsx]: 1

File: src/Styleguide/Components/AuctionResultItem.tsx [hunk 7]
```diff
@@ -0,0 +1,183 @@
...
+export const SmallAuctionResultItem = (props: AuctionResultItemProps) => {
+  return (
+    <Flex width="100%" justifyContent="space-between">
+      <Flex width="70%">
+        <div>
+          <ScaledArtworkImage
+            src={props.imageUrl}
+            width="auto"
+            height="70px"
+            mx={4}
```
Comment (damassi) [inline src/Styleguide/Components/AuctionResultItem.tsx]: 2

File: src/Styleguide/Components/AuctionResultItem.tsx [hunk 8]
```diff
@@ -0,0 +1,183 @@
...
+        <div>
+          <Serif size="2">
+            <Title>{props.title}, </Title>
+            {props.date}
+          </Serif>
+          <MetadataContainer>
+            <Serif size="2">{props.dimensions.in}</Serif>
+            <Serif size="2">{props.dimensions.cm}</Serif>
+          </MetadataContainer>
+          <Box pt={3}>
```
Comment (damassi) [inline src/Styleguide/Components/AuctionResultItem.tsx]: 1

File: src/Styleguide/Components/AuctionResultItem.tsx [hunk 9]
```diff
@@ -0,0 +1,183 @@
...
+export const LargeAuctionResultItem = (props: AuctionResultItemProps) => {
+  return (
+    <Flex width="100%" justifyContent="space-between">
+      <Flex width="40%">
+        <div>
+          <ScaledArtworkImage
+            src={props.imageUrl}
+            width="auto"
+            height="70px"
+            mx={4}
```
Comment (damassi) [inline src/Styleguide/Components/AuctionResultItem.tsx]: 2

**Comment by damassi** [comment]
@mzikherman - once those values are updated feel free to merge 👍

**Pull request merged**